README.md

```diff
@@ -40,19 +40,19 @@
 
 We love the default GitHub profiles and we want to enhance them:
 
-* The GitHub profiles aren't clearly showing all repos you have contributed to since you joined
+* The GitHub profiles clearly aren't showing all the repos you have contributed to since you joined
```

Comment:
1. aren't clearly showing them = they show them, but not in a clear manner
2. clearly aren't showing them = they obviously don't show them at all

I do mean number 1 :)